src/entrypoint/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:latest
+WORKDIR /root/
+COPY *.go ./
+RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o app .
+
+FROM alpine:latest
```

Comment:
You may safely use 'FROM scratch' here, as it seems like application has no dependencies for external libraries.